docs/guided-tour/README.md

```diff
@@ -25,20 +27,12 @@ sidebar_position: 1
   It is possible that some or all of these clusters coincide, e.g. in the most simplistic approach, you have only one
   cluster. Such a "one-cluster-setup" is the easiest way to start working with the Landscaper.
 
-> **_NOTE:_** The Landscaper now also supports [OCM (Open Component Model)](https://ocm.software/) Component
-> Descriptors [Version 3](https://ocm.software/docs/component-descriptors/version-3/), additionally to [Version
-> 2](https://ocm.software/docs/component-descriptors/version-2/).  
-> Since we try our best to avoid disruptions, this functionality is currently behind a feature switch. For detailed 
-> information on how to enable this for your own landscaper instance, set the corresponding flag in the configuration to
-> `true`, as shown [here](https://github.com/gardener/landscaper/blob/master/docs/installation/install-landscaper-controller.md#configuration-through-valuesyaml).
-> If you just want to follow along the tour, you should have the switch enabled!
-
 ## How to follow the Tour
 
 The Guided Tour consists of several chapters, some of which build on each other. In order to get the most out of it, 
-you should be following the Guided Tour in this sequence. Another important point is, that this documentation
-is not optimal with respect to getting an overview about the used example resources in the Guided Tour. Therefore,
-it might be better to read it in the [original github repository](https://github.com/gardener/landscaper/tree/master/docs/guided-tour).
+you should be following the Guided Tour in this sequence. The Guided Tour is not presented very well
+if preprocessed with some documentation tooling. It might be better to read it with your browser in the 
+[original github repository](https://github.com/gardener/landscaper/tree/master/docs/guided-tour).
```

Comment:
I don't really understand this. Who is preprocessing with some documentation tooling? How is this impacting the Landscaper user?

Reply:
We have removed this sentence.